pkg/apis/v1alpha1/register.go

```diff
@@ -38,26 +38,45 @@ var (
 	RestrictedLabelDomains = []string{
 		LabelDomain,
 	}
-	AMIFamilyBottlerocket = "Bottlerocket"
-	AMIFamilyAL2          = "AL2"
-	AMIFamilyUbuntu       = "Ubuntu"
-	AMIFamilyCustom       = "Custom"
-	SupportedAMIFamilies  = []string{
+	AMIFamilyBottlerocket  = "Bottlerocket"
+	AMIFamilyAL2           = "AL2"
+	AMIFamilyUbuntu        = "Ubuntu"
+	AMIFamilyWindowsServer = "WindowsServer"
+	AMIFamilyCustom        = "Custom"
+	SupportedAMIFamilies   = []string{
 		AMIFamilyBottlerocket,
 		AMIFamilyAL2,
 		AMIFamilyUbuntu,
+		AMIFamilyWindowsServer,
 		AMIFamilyCustom,
 	}
 	SupportedContainerRuntimesByAMIFamily = map[string]sets.String{
-		AMIFamilyBottlerocket: sets.NewString("containerd"),
-		AMIFamilyAL2:          sets.NewString("dockerd", "containerd"),
-		AMIFamilyUbuntu:       sets.NewString("dockerd", "containerd"),
+		AMIFamilyBottlerocket:  sets.NewString("containerd"),
+		AMIFamilyAL2:           sets.NewString("dockerd", "containerd"),
+		AMIFamilyUbuntu:        sets.NewString("dockerd", "containerd"),
+		AMIFamilyWindowsServer: sets.NewString("containerd"),
 	}
-	ResourceNVIDIAGPU             v1.ResourceName         = "nvidia.com/gpu"
-	ResourceAMDGPU                v1.ResourceName         = "amd.com/gpu"
-	ResourceAWSNeuron             v1.ResourceName         = "aws.amazon.com/neuron"
-	ResourceHabanaGaudi           v1.ResourceName         = "habana.ai/gaudi"
-	ResourceAWSPodENI             v1.ResourceName         = "vpc.amazonaws.com/pod-eni"
+
+	Windows2019              = "2019"
```

Comment:
Is it possible to introduce a single windows version as the default windows version and then build off of that once we get the initial version out and tested by users?